sweepbatcher/sweep_batcher.go

```diff
@@ -253,6 +253,13 @@ type Batcher struct {
 	// exit.
 	wg sync.WaitGroup
 
+	// waitingPeriod is the delay of first batch publishing after creation.
+	// It only affects newly created batches, not batches loaded from DB,
+	// so publishing does happen in case of a crashloop. If a sweep is about
```

Comment:
Done. It is important to mention a crashloop, because it is specifically dangerous if we wait (and restart again before finishing waiting). So I mentioned both restarts and crashloop in parenthesis.